.trunk/trunk.yaml

```diff
@@ -45,7 +45,7 @@ lint:
       commands:
         - name: lint
           output: sarif
-          run: pyright --project .trunk/pyrightconfig.backend-api.json --outputjson ${target}
+          run: pyright --project .trunk/pyrightconfig.backend-api.json --outputjson
```

Comment:
This was removed as it was telling `pyright` to scan the whole directory, effectively overriding the `excludes`/`includes` in the config